[DOCS-13265] Add guide for creating monitors for sensitive data (#34296)
DataDog/documentation: content/en/security/sensitive_data_scanner/guide/create-monitors-for-sensitive-data.md

---
title: Create Monitors for Sensitive Data

> description: Best practices for creating monitors to alert on sensitive data matches detected by Sensitive Data Scanner across logs, APM traces, and RUM events.

Reviewer: ```suggestion
description: Best practices for creating monitors that alert on sensitive data matches detected by Sensitive Data Scanner across logs, APM traces, and RUM events.
```

further_reading:
  - link: /security/sensitive_data_scanner/
    tag: Documentation
    text: Set up Sensitive Data Scanner
  - link: /security/sensitive_data_scanner/scanning_rules/library_rules
    tag: Documentation
    text: Learn more about out-of-the-box library rules
  - link: /monitors/types/log/
    tag: Documentation
    text: Log monitors
  - link: /monitors/types/apm/
    tag: Documentation

>     text: APM monitors

Reviewer: ```suggestion
    text: Learn to set up APM monitors
```

---
## Overview

After setting up Sensitive Data Scanner to detect and redact sensitive information in your telemetry data, you can create monitors to alert when sensitive data matches are detected. This helps you:

- Detect when sensitive data is being logged or transmitted unexpectedly
- Alert security and compliance teams to potential data exposure
- Track sensitive data volume trends over time
- Identify services or teams that need additional security guidance

> When Sensitive Data Scanner detects a match, it automatically adds tags to the data that you can use to create targeted monitors. This guide shows you how to create monitors for sensitive data across logs, APM traces, and RUM events.

Reviewer: ```suggestion
When Sensitive Data Scanner detects a match, it automatically adds tags to the data that you can use to create targeted monitors. This guide shows you how to create monitors to alert on sensitive data detected across logs, APM traces, and RUM events.
```
